app/templates/gulpfile.js

```diff
@@ -165,15 +165,19 @@ gulp.task('inject', ['wiredep', 'styles', 'templatecache'], function() {
 
 
 /**
- * Creates a sample gulp-local.json; can be used as model for other environments
+ * Creates a sample local.json; can be used as model for dev.json and prod.json
  */
 gulp.task('init-local', function() {
   //copy from slushfile - config gulp - with modifications to use config instead
-  log('Creating gulp-local.json sample document with values drawn from gulp.config.js');
+  log('Creating local.json sample document with values drawn from gulp.config.js');
   try {
     var configJSON = {};
     configJSON['ml-version'] = config.marklogic.version;
     configJSON['ml-host'] = config.marklogic.host;
+    configJSON['ml-admin-user'] = config.marklogic.username;
+    configJSON['ml-admin-pass'] = config.marklogic.password;
+    configJSON['ml-app-user'] = config.marklogic.username;
```

Comment:
I think we could/should go the other way around. Name of default-user is predictable, so we can deduce that (or we could just replace it), and we could generate a new password, and override the one generated by Roxy..